tests/integration/courses.test.js

```diff
@@ -1851,6 +1852,21 @@ describe('COURSES ROUTES - PUT /courses/{_id}', () => {
       expect(response.statusCode).toBe(403);
     });
 
+    it(
+      'should return 200 if holding admin try to update contact and company representative which is contact',
```

Comment:
tu peux enlever le `try to` juste mettre `updates`
et tu peux enlever `contact and` aussi puisque tu remets a la fin `which is contact`

"should return 200 if holding admin updates company representative which is contact" c'est suffisant non ?

Reply:
done